### PR TITLE
Wdp191203-6

### PR DESCRIPTION
**Pull request by kacperkoza34**
Problem: 
No info about old price in ProductBox

Solution:
Add oldPrice in initialState, if oldPrice is equal to 0 there is no oldPrice, add conditional statement in JSX

### CODE REVIEW

**Comment by Varenthein** [comment]
Where's the PR description? As I said at the meeting, there should be clear information about the what problem you are trying to solve with this task and how you've done it.

File: src/components/common/ProductBox/ProductBox.js [hunk 1]
```diff
@@ -49,7 +49,8 @@ const ProductBox = ({ name, price, promo, stars, image }) => (
       </div>
       <div className={styles.price}>
         <Button noHover variant='small'>
-          $ {price}
+          {oldPrice != 0 && <span className={styles.oldPrice}>$ {oldPrice} </span>}${' '}
```
Comment (Varenthein) [inline src/components/common/ProductBox/ProductBox.js]: Hmm... Why don't we just use `oldPrice` as condition? `{oldPrice && <span className={styles.oldPrice}>$ {oldPrice} </span>}${' '}`. It should work the same do't you think? :)

Comment (kacperkoza34) [reply src/components/common/ProductBox/ProductBox.js]: that was my first idea but it dosen't work the same 
contidional statment

Comment (kacperkoza34) [reply src/components/common/ProductBox/ProductBox.js]: i think we can leave it as it is

**Comment by OskarLebuda** [comment]
Please resolve all conflicts.

File: src/components/common/ProductBox/ProductBox.js (hunk 1 above)
Comment (Varenthein) [reply src/components/common/ProductBox/ProductBox.js]: Okey. So let it be. Resolve conflicts and we are good to go :)

**Pull request merged**